config/crl_config.go

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
rename the file to crlConfig.go to be consistent.

Reply:
Sure.